lms/static/scripts/frontend_apps/components/dashboard/SyncGradesButton.tsx

```diff
@@ -20,10 +20,18 @@ export type SyncGradesButtonProps = {
    * Passing `undefined` means students are not yet known and/or being loaded.
    */
   studentsToSync?: Array<{ h_userid: string; grade: number }>;
+
+  /**
+   * Invoked after a sync was scheduled successfully.
```

Comment:
What happens if the sync is not scheduled successfully? Is that scenario handled internally within the component?

Reply:
> Is that scenario handled internally within the component?

Yes. In that case the last sync is marked as error, which will put the button in a "Sync failed. Click to retry" state.

https://github.com/hypothesis/lms/pull/6712/files#diff-581f6b4dde8f047b2629b55a1a0730a8509f31990880fb065ddfc193a283637eL117